templates/pwm.asg.element.json

```diff
@@ -0,0 +1,884 @@
+                                            {
+                                                "Ref": "AWS::Region"
+                                            },
+                                            "\n",
+                                            "runas=root\n"
+                                        ]
+                                    ]
+                                },
+                                "group": "root",
+                                "mode": "000400",
+                                "owner": "root"
+                            }
+                        },
+                        "services": {
+                            "sysvinit": {
+                                "cfn-hup": {
+                                    "enabled": "true",
+                                    "ensureRunning": "true",
+                                    "files": [
+                                        "/etc/cfn/cfn-hup.conf",
+                                        "/etc/cfn/hooks.d/cfn-auto-reloader.conf"
+                                    ]
+                                }
+                            }
+                        }
+                    },
+                    "b-cloudwatchagentsetup": {
+                        "commands": {
+                            "10-install-unzip": {
+                              "command": "sudo yum install unzip -y"
+                            },
+                            "20-get-agent": {
+                                "command": "wget https://s3.amazonaws.com/amazoncloudwatch-agent/linux/amd64/latest/AmazonCloudWatchAgent.zip"
+                            },
+                            "30-unpack-agent": {
+                                "command": "unzip AmazonCloudWatchAgent.zip -d /usr/local/bin"
+                            },
+                            "40-install-agent": {
+                                "command": "cd /usr/local/bin && bash -xe /usr/local/bin/install.sh"
+                            },
+                            "50-enable-agent": {
+                                "command": "systemctl enable amazon-cloudwatch-agent.service"
+                            },
+                            "60-start-agent": {
+                                "command": "systemctl start amazon-cloudwatch-agent.service"
+                            },
+                            "70-get-conf": {
+                                "command": {
+                                    "Fn::Join": [
+                                        "",
+                                        [
+                                            "aws s3 cp s3://",
+                                            {
+                                                "Ref": "ConfigBucketName"
+                                            },
+                                            "/pwmcloudwatchagent.json /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json"
```

Comment:
Where is the `pwmcloudwatchagent.json` file in source control? How does it get to the s3 bucket?

Reply:
Ok, then i think 70-get-conf can be removed?